src/averageAges.js

```diff
@@ -15,11 +15,11 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const isMan = findSex(people, 'm');
```

Comment:
`isMan` makes it look like this variable indicates boolean, however, that's an array, so you'd rather call it `men`